modules/sc-mesh-secure-deployment/src/nats/scripts/cli_settings_request.py

```diff
@@ -3,27 +3,80 @@
 import json
 import config
 
-
 async def main():
     # Connect to NATS!
     nc = await client.connect_nats()
-    cmd_dict = {"api_version": 1, "ssid": "test_mesh", "key": "1234567890",
-                "ap_mac": "00:11:22:33:44:55", "country": "FI", "frequency": "5220",
-                "frequency_mcc": "2412", "routing": "batman-adv", "priority": "long_range",
-                "ip": "192.168.1.2", "subnet": "255.255.255.0", "tx_power": "5", "mode": "mesh",
-                "role": f"{config.MODULE_ROLE}"}
+    cmd_dict = {
+        "api_version": 1,
+        "role": f"{config.MODULE_ROLE}",  # sleeve, drone, gcs
+        "radios": [
+            {
+                "radio_index": "0",
+                "ssid": "test_mesh2",
+                "key": "1234567890",
+                "ap_mac": "00:11:22:33:44:55",
+                "country": "US",  # all radios must have the same country
+                "frequency": "2412",
+                "frequency_mcc": "2412",  # multiradio not supporting
+                "routing": "batman-adv",
+                "priority": "long_range",
+                "ip": "10.20.15.3",
+                "subnet": "255.255.255.0",
+                "tx_power": "15",
+                "mode": "mesh",  # ap+mesh_scc, mesh, halow
+                "mesh_vif": "wlp2s0",
+                "batman_iface": "bat0",
+            },
+            {
+                "radio_index": "1",
+                "ssid": "test_mesh",
+                "key": "1234567890",
+                "ap_mac": "00:11:22:33:44:55",
+                "country": "US",  # all radios must have the same country
+                "frequency": "5220",
+                "frequency_mcc": "2412",  # multiradio not supporting
+                "routing": "batman-adv",
+                "priority": "long_range",
+                "ip": "10.20.15.3",
+                "subnet": "255.255.255.0",
+                "tx_power": "15",
+                "mode": "mesh",  # ap+mesh_scc, mesh, halow
+                "mesh_vif": "wlp3s0",  # this needs to be correct
+                "batman_iface": "bat0",
+            },
+            {
+                "radio_index": "2",
+                "ssid": "test_mesh3",
+                "key": "1234567890",
+                "ap_mac": "00:11:22:33:44:55",
+                "country": "US",  # all radios must have the same country
+                "frequency": "5190",
+                "frequency_mcc": "2412",  # multiradio not supporting
+                "routing": "batman-adv",
+                "priority": "long_range",
+                "ip": "10.20.15.3",
+                "subnet": "255.255.255.0",
+                "tx_power": "30",
+                "mode": "halow",  # ap+mesh_scc, mesh, halow
+                "mesh_vif": "halow1",
+                "batman_iface": "bat0",
```

Comment:
Selina's work still needs to be merged into this CommsController but just in case it helps to mention now, in short:

1. Each interface (11s WiFi, STA WiFi, RJ45, USB, etc) such as halow1 here will be given to the Network Access Control logic
2. NAC will detect peers (with help from MESH-PEER-CONNECTED notifications on the wpa_supplicant control interface wherever possible) and setup a "lower MACsec" interface for each one of them, for example lms002091abcdef to for the secure channel between here and the 00:20:91:ab:cd:ef interface on the other side.
3. Each new lms* interface gets added to the lower MACsec bridge called br-lms for example, which filters (a lot like "client isolation" in a WiFi AP) to allow traffic only between each lms* and the veth uplink explained next, as opposed to allowing traffic between two lms* interfaces which would create a network loop otherwise.
4. Adding batman-adv directly on top on br-lms (not to be confused with how so far we have been adding a bridge on top of a batman-adv interface like bat0 as one of the downlinks of br-lan) would make it a bit difficult to write the filtering rules from the previous point, so we add a "veth" imaginary cable: one side is given to the switch like `brctrl addif br-lms veth-foo` and the other side is given to batman-adv like `batctl [meshif bat0] add if veth-bar`.
5. Steps 2, 3 & 4 are repeated for what we call the "upper" part which results in things like ums002091abcdef, br-ums, bat1, etc.
6. NATS Provisioning etc (e.g. OS updates in the future) can use bat1.

Reply:
Yes, but with separate PR.  This is for multiradio implementation.